README.md

```diff
@@ -57,12 +57,12 @@ Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 L
       - [MicroG](https://microg.org)
         - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
       - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
+    - or you can just search the web or the [xda-developers forum](https://xdaforums.com/) for an appropriate version for your device.
 3. Start the desktop app and follow the instructions.
 
 ## Officially supported devices
 
-Currently, the **we support 73 devices** by various vendors and working on adding more soon!
+Currently, the **we support 74 devices** by various vendors and working on adding more soon!
```

Comment:
Did you recount it? Maybe I made an error there, then I should also update the website.

Reply:
nope, added a7 2017 support, so I thought Id just increase it by one
It is essentially the same as the a5 2017, so I just pasted the configs and called it a day